front-end/src/main/resources/static/index.html

```diff
@@ -150,41 +149,38 @@ <h1>User Profile:
     });
   }
 
-  function doInit() {
+  async function doInit() {
 
-    $( "#logout" ).click(function() {
-      oktaSignIn.signOut(() => {
-        oktaSignIn.tokenManager.clear();
+    $("#logout").click(function () {
+        oktaSignIn.authClient.tokenManager.clear();
         location.reload();
-      });
     });
 
     // Check if we already have an access token
-    const token = oktaSignIn.tokenManager.get('my_access_token');
+    const token = await oktaSignIn.authClient.tokenManager.get('accessToken');
 
     // if we do great, just go with it!
     if (token) {
-      doAllTheThings(token.accessToken)
+      doAllTheThings(token.value)
     } else {
-
       // otherwise show the login widget
       oktaSignIn.renderEl(
         {el: '#okta-login-container'},
-        function (response) {
-
-          // check if success
-          if (response.status === 'SUCCESS') {
 
-            // for our example we have the id token and the access token
-            oktaSignIn.tokenManager.add('my_id_token', response[0]);
-            oktaSignIn.tokenManager.add('my_access_token', response[1]);
+        function success(res) {
+          // OIDC response
 
-            // hide the widget
-            oktaSignIn.hide();
+          // If the widget is configured for OIDC with multiple responseTypes, the
+          // response will be an array of tokens:
+          // i.e. authParams.responseType = ['id_token', 'token']
+          oktaSignIn.authClient.tokenManager.add('idToken', res.tokens.idToken);
+          oktaSignIn.authClient.tokenManager.add('accessToken', res.tokens.accessToken);
```

Comment:
@swiftone Is there a default success handler that would do this for me?

Reply:
Not that I'm aware of - the widget hands off to external code to manage the tokens.  Our